app/models/repository_cell.rb

```diff
@@ -8,6 +8,10 @@ class RepositoryCell < ActiveRecord::Base
   validates :repository_row,
             uniqueness: { scope: :repository_column }
 
+  belongs_to :repository_text_value, optional: true, foreign_key: :value_id
+  belongs_to :repository_date_value, optional: true, foreign_key: :value_id
+  belongs_to :repository_list_value, optional: true, foreign_key: :value_id
+
```

Comment:
This was updated with recent commits, so please don't overwrite it when merging

Reply:
:+1: